app/dcs/third-party-tools/veeam/page.md

```diff
@@ -206,11 +202,11 @@ Use the New **Backup Job wizard** to configure the backup job. Follow the steps
 
    ![](https://link.storjshare.io/raw/jua7rls6hkx5556qfcmhrqed2tfa/docs/images/veeam1.png)
 
-3. Upon opening **Advanced** you will be presented with the option of selecting **Storage Optimization**. **Large** (4MB) and **extra large blocks** (8MB) are the ideal settings for best performance and least segment cost. **Extra large** (8MB) is the ideal setting but is hidden by default. 
+3. Upon opening Advanced - Storage, you will be presented with the option of selecting Storage Optimization. The ideal setting for object storage is **4MB** or **8MB**.  The larger sizes provide faster backup/restores and lowers the [Storj segment cost](https://docs.storj.io/dcs/pricing#per-segment-fee). Note: using 4MB or 8MB will result in larger incremental backups compared to 1MB (the Veeam default), but Storj's recommendation is to choose 4MB or 8MB to dramatically reduce your Storj costs while also providing better backup and restore times.
```

Comment:
One gotcha here, internal links to the docs site have a unique format, so instead of using the url, use the docId (see in the front matter of the page https://github.com/storj/docs/blob/main/app/dcs/pricing/page.md

So for it to build properly, change the link to:
```
[Storj segment cost](docId:59T_2l7c1rvZVhI8p91VX#per-segment-fee)
```

Reply:
This has been changed.  Thanks for the guidance, @amozoss !